### DISCUSSION TITLE
Afraid Zorin install will wipe out my macOS

### DISCUSSION THREAD

**Discussion by ekleinberg** [utmapp/UTM#6278] [UTM Help (macOS)]
<img width="1392" alt="Screenshot 2024-04-15 at 4 35 41 PM" src="https://github.com/utmapp/UTM/assets/149612606/af1c6226-73ea-44f8-b4dc-b15b25ddbd6e">

I am installing Zorin on my mac mini running Sonoma and using the UTM virtualization software. I have included the screenshot which concerns me. The way it is worded makes me worried my mac mini might be wiped clean if I continue. I hope it means hat only the virtual drive it creates will be wiped. Can you please confirm one way or another? Thanks!

**Comment by Samuel525432** [comment]
> <img width="1392" alt="Screenshot 2024-04-15 at 4 35 41 PM" src="https://github.com/utmapp/UTM/assets/149612606/af1c6226-73ea-44f8-b4dc-b15b25ddbd6e">
> 
> 
> 
> I am installing Zorin on my mac mini running Sonoma and using the UTM virtualization software. I have included the screenshot which concerns me. The way it is worded makes me worried my mac mini might be wiped clean if I continue. I hope it means hat only the virtual drive it creates will be wiped. Can you please confirm one way or another? Thanks!


The virtual drive will be wiped but not your main drive 

I used to test Linux distros like these on my M3 MacBook Air
